### PR TITLE
Remove information from README.md that is ASF-specific

### PR DESCRIPTION
**Pull request by forrestfwilliams**

### CODE REVIEW

**Comment by forrestfwilliams** [comment]
@jhkennedy two question for you:

1. Since we no longer talk about ECR, where and how should we talk about the GitHub Container Registry? Is publishing handled by an action, and what should the user know about container publishing?

2. Does ASFHyP3/Automation need to be able to push to branches?

File: README.md [hunk 1]
```diff
@@ -67,9 +65,9 @@ We want to push the local copy we just created to our GitHub repository:
 ```bash
 # From hyp3-<process>
 git init .
-git remote add origin git@github.com:ASFHyP3/hyp3-<process>.git
+git remote add origin git@github.com:YOUR-GITHUB-USERNAM-OR-ORG-NAME/hyp3-<process>.git
```
Comment (jhkennedy) [inline README.md:68]:
```suggestion
git remote add origin git@github.com:<GITHUB_USERNAME_OR_ORG>/hyp3-<process>.git
```

File: README.md [hunk 2]
```diff
@@ -7,13 +7,7 @@ generate a new HyP3 Plugin.
...
 ### 0. Create a repository on GitHub
 
-To create a new plugin, you'll first need to create a new HyP3 repository in ASF's
-GitHub:
-
-* https://github.com/organizations/ASFHyP3/repositories/new
-
-*Note: If you don't have repo create permissions in the `ASFHyP3` org, ask the
-  Tools team to create you a repo!*
+To create a new plugin, you'll first need to create a new repository on [GitHub](https://github.com) where you will contain your plugin.
```
Comment (jhkennedy) [inline README.md:10]:
```suggestion
To create a new plugin, you'll first need to create a new repository on [GitHub](https://github.com) for your plugin.
```

File: README.md [hunk 3]
```diff
@@ -94,49 +92,46 @@ Now, go back to the development branch:
...
-       * click "Restrict who can push to matching branches"
-         * add "ASFHyP3/automation" to who can push
+       * click "Do not allow bypassing of the above settings"
        * Create
 
-### 6. Restart the GitHub Actions
+For both the `main` and `develop` you can restrict who can push to the branch.
+In the same page where you set the above options, you can also click "Restrict
+who can push to matching branches", then search and add the desired people/organizations
+who are allowed to push.
```
Comment (jhkennedy) [inline README.md:128]: Right now, the release workflow does assume that whatever account the access token is associated with has the ability to push to `develop`.

File: README.md [hunk 4]
```diff
@@ -94,49 +92,46 @@ Now, go back to the development branch:
...
+
+For more information on how to contribute to repositories set up in this manner,
+check out GitHub's [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+article
```
Comment (jhkennedy) [inline README.md:132]: fine for now, I think. But we really should write down in outrwiki (or somewhere publically accessible) how our "flow" works

File: README.md [hunk 5]
```diff
@@ -94,49 +92,46 @@ Now, go back to the development branch:
...
-
-Create a docker repository for your plugin in the `hyp3-v2-full-access` account:
-   ```bash
-   # Assuming your aws cli is setup to use the hyp3-v2-full-access profile
-   aws ecr create-repository \
-       --repository-name hyp3-<process> \
-       --image-scanning-configuration scanOnPush=true \
-       --region us-west-2
-   ```
+### 4. Configure the GitHub repository settings
```
Comment (jhkennedy) [inline README.md:95]: Might need to add a section on configuring GHCR. One point to make:
* "packages" (here, container images) are always private by default, so they'll need to be made public explicitly after the first image is pused.

**Comment by jhkennedy** [comment]
@forrestfwilliams I think I answered in my review -- we can chat about the specifics

File: README.md (hunk 5 above)
Comment (forrestfwilliams) [reply README.md:95]: I could use some help fleshing out a section on the GHCR. Not sure I understand how it works!

**Comment by forrestfwilliams** [comment]
Thanks @jhkennedy! Does ASFHyP3/Automation need to be able to push to the plugins's branches?

File: README.md (hunk 5 above)
Comment (jhkennedy) [reply README.md:95]: Let's actually punt talking about GHCR until we use this for the hyp3-isce plugin, so we know exactly what steps we'll need to take

File: README.md [hunk 6]
```diff
@@ -7,13 +7,7 @@ generate a new HyP3 Plugin.
...
-To create a new plugin, you'll first need to create a new HyP3 repository in ASF's
-GitHub:
-
-* https://github.com/organizations/ASFHyP3/repositories/new
-
-*Note: If you don't have repo create permissions in the `ASFHyP3` org, ask the
-  Tools team to create you a repo!*
+To create a new plugin, you'll first need to create a new repository on [GitHub](https://github.com) for your plugin.
   
 You should enter a repository name like `hyp3-<process>` where `<process>` is the
```
Comment (jhkennedy) [inline README.md:12]:
```suggestion
You should enter a repository name like `hyp3-<PROCESS>` where `<PROCESS>` is the 
```

File: README.md [hunk 7]
```diff
@@ -67,9 +65,9 @@ We want to push the local copy we just created to our GitHub repository:
 ```bash
 # From hyp3-<process>
 git init .
-git remote add origin git@github.com:ASFHyP3/hyp3-<process>.git
+git remote add origin git@github.com:<GITHUB_USERNAME_OR_ORG>/hyp3-<process>.git
```
Comment (jhkennedy) [inline README.md:68]:
```suggestion
git remote add origin git@github.com:<GITHUB_USERNAME_OR_ORG>/hyp3-<PROCESS>.git
```

File: README.md [hunk 8]
```diff
@@ -94,50 +92,63 @@ Now, go back to the development branch:
...
+container for your process.
+
+### 6. Make HyP3 plugin container public
+
+Once the Actions have successfully run, a containerized version of your plugin will be
+available in the GitHub Container Registry (GHCR). You can find this plugin in the "Packages"
+section of your GitHub user/organization account. You can also `pull` it to your local
+machine for use using the command:
+
+`docker pull ghcr.io/GH_ACCOUNT_NAME/GH_REPOSITORY_NAME`
```
Comment (jhkennedy) [inline README.md:148]:
```suggestion
`docker pull ghcr.io/<GH_ACCOUNT_NAME>/<GH_REPOSITORY_NAME>`
```

**Review by jhkennedy — APPROVED** [review]
Looks good! We'll revisit as we use the cookie-cutter for hyp3-isce.

Just a few nitpicky suggestions.

**Pull request merged**